tests/runner/parent_main.js

```diff
@@ -9,13 +8,15 @@ require(['config'], function() {
 
             var tests = [
                 //'../../tests/unit/constructor.js',
```

Comment:
BTR has a method that uses [require glob to bring the files in](https://github.com/mobify/btr-adaptivejs/blob/development/tests/runner/testRunner.js#L15). Can we swap to using that?